recipe/build-mpi.sh

```diff
@@ -20,6 +20,14 @@ if [ $(uname) == Darwin ]; then
     export LDFLAGS="$LDFLAGS -Wl,-rpath,$PREFIX/lib"
 fi
 
+# At this point, we selected the correct docker image, but we have no
+# way to know tell if CUDA is present, so we look for nvcc
+if [ $(command -v nvcc) ]; then
```

Comment:
Conda-build passes through all the variants into the build environment. So we can just check this directly.

```suggestion
if [ ${cuda_compiler_version} != "None" ]; then
```

Reply:
Oh cool, thanks @jakirkham. I didn’t know this. It’s much better.